wix/main.wxs

```diff
@@ -0,0 +1,222 @@
+                            Id='npmEXE'
+                            Name='npm.exe'
+                            DiskId='1'
+                            Source='target\release\npm.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                    <Component Id='npxBinary' Guid='*' Win64='$(var.Win64)'>
+                        <File
+                            Id='npxEXE'
+                            Name='npx.exe'
+                            DiskId='1'
+                            Source='target\release\npx.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                    <Component Id='yarnBinary' Guid='*' Win64='$(var.Win64)'>
+                        <File
+                            Id='yarnEXE'
+                            Name='yarn.exe'
+                            DiskId='1'
+                            Source='target\release\yarn.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                </Directory>
+            </Directory>
+            <Component Id='UserBinDir' Guid='96B83015-59EC-42E7-B85E-BDF6A1BC6198' Win64='$(var.Win64)'>
+                <RegistryValue
+                    Root='HKCU'
+                    Key='Software\[Manufacturer]\[ProductName]'
+                    Name='BinDir'
+                    Type='string'
+                    Value='[LocalAppDataFolder]Notion\bin'
+                    KeyPath='yes' />
+                <Environment
+                    Id='BINPATH'
+                    Name='PATH'
+                    Value='[LocalAppDataFolder]Notion\bin'
+                    Permanent='no'
+                    Part='first'
+                    Action='set'
+                    System='no' />
+            </Component>
+        </Directory>
+
+        <Feature Id='MainProgram'>
+            <ComponentRef Id='InstallDir'/>
+            <ComponentRef Id='UserBinDir'/>
+            <ComponentRef Id='notionBinary'/>
+            <ComponentRef Id='launchbinBinary'/>
+            <ComponentRef Id='launchscriptBinary'/>
+            <ComponentRef Id='nodeBinary'/>
+            <ComponentRef Id='npmBinary'/>
+            <ComponentRef Id='npxBinary'/>
+            <ComponentRef Id='yarnBinary'/>
+        </Feature>
+
+        <SetProperty Id='ARPINSTALLLOCATION' Value='[APPLICATIONFOLDER]' After='CostFinalize'/>
+
+        
+        <!--
+          Uncomment the following `Icon` and `Property` tags to change the product icon.
+
+          The product icon is the graphic that appears in the Add/Remove
+          Programs control panel for the application.
+        -->
+        <!--<Icon Id='ProductICO' SourceFile='wix\Product.ico'/>-->
+        <!--<Property Id='ARPPRODUCTICON' Value='ProductICO' />-->
+
+        <Property Id='ARPHELPLINK' Value='https://github.com/notion-cli/notion'/>
+        <Property Id='LicenseAccepted' Value='1'/>
+
+        <UI>
+            <UIRef Id='WixUI_FeatureTree' />

```

Comment:
You only specify a single Feature, I feel the FeatureTree Ui might be a bit of an overkill. Any reason you chose this one?

Reply:
I initially started with the `WixUI_Minimal`, but I wanted to have the initial splash screen, instead of the first screen just being the EULA page. `WixUI_FeatureTree` was the most minimal that still included the `WelcomeDlg`, so I started there and then used the `Publish` features to skip over the actual feature selection screen (Since, as you say, we only have one feature and it's not optional). It also will make it smoother if we update in the future to have more features and optional ones, since the remaining screens will stay the same.

Reply:
Legit!